tests/S3/ObjectCopierTest.php

```diff
@@ -111,26 +112,6 @@ function (CommandInterface $cmd, RequestInterface $req) {
                         );
                         break;
 
-                    case 'UploadPartCopy':
```

Comment:
Why are we removing this case?

Reply:
Technically I am not removing this case, instead I am just combining it with the "CopyObject" case. The reason for that is because they both do the same.

Reply:
Ah, I skimmed over lines 84-88 because they were collapsed.  That makes sense to me